api/config/rolepermission.php

```diff
@@ -0,0 +1,521 @@
+<?php
+
+return [
+    /*
+    |--------------------------------------------------------------------------
+    | Actions
+    |--------------------------------------------------------------------------
+    |
+    | A map of all actions available within the application.
+    |
+    */
+    'actions' => [
+        'v' => 'view',
+        'c' => 'create',
+        'u' => 'update',
+        'd' => 'delete',
+        'as' => 'assign',
+        'arc' => 'archive',
+        's' => 'submit',
+        'p' => 'publish'
+    ],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Scopes
+    |--------------------------------------------------------------------------
+    |
+    | A map of all scopes for different actions performed.
+    |
+    */
+    'scopes' => [
+        'o' => 'own',
+        'a' => 'any',
+        't' => 'team'
+    ],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Resources
+    |--------------------------------------------------------------------------
+    |
+    | A map of all resources that a specific action can be taken on.
+    |
+    */
+    'resources' => [
```

Comment:
Would it not make sense to use the same number of characters for each key for consistency? Maybe at least three characters in case another action is added in the future.